MetaMask/metamask-extension
# fix: MMASSETS-475 - Add network name to asset details page

## **Description**

Adds the network name to the assets details page so the user can be sure to know what network a given asset lives on.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/29211?quickstart=1)

## **Related issues**

Fixes: https://consensyssoftware.atlassian.net/browse/MMASSETS-475

## **Manual testing steps**

1. Click any number of asset items in the asset list
2. Ensure that the network name that displays matches the asset network and images

## **Screenshots/Recordings**



### **Before**



### **After**

<img width="748" alt="SCR-20241213-osij" src="https://github.com/user-attachments/assets/4811d56f-b856-4cef-af0f-e7ac696a4c60" />



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
